api/dependencies/auth.py

```
@@ -96,7 +107,9 @@ async def verify_token(credentials: HTTPAuthorizationCredentials = Depends(secur
         return cast(Dict[Any, Any], decoded_token)
     except Exception as e:
         logger.error(f"Invalid token: {str(e)}")
-        raise HTTPException(status_code=401, detail=f"Invalid authentication credentials: {str(e)}")
+        raise HTTPException(
+            status_code=401, detail=f"Invalid authentication credentials: {str(e)}"
+        )
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)